src/modules/sign/state/GithubLoginContext.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useReducer } from "react";
```

Comment:
I wonder why there is a `GithubLoginContext` 🤔 . Would there be a `GitlabLoginContext`? Why not just have a generic `AuthContext` that can handle both Github and Gitlab authentication?